docs/jasmine-options.md

```diff
@@ -112,6 +112,12 @@ Default: `'full'`
   * `short` only displays a success or failure character for each test (useful with large suites)
   * `none` displays nothing
 
+## options.showLogging
+Type: `Boolean`
+Default: `false`
+
+Display console logging messages while running tests
+
```

Comment:
This option should probably be added to the options list with an explicit default value.